common/functions/src/scalars/expressions/cast_test.rs

```diff
@@ -38,7 +38,7 @@ fn test_cast_function() -> Result<()> {
             display: "CAST",
             nullable: false,
             columns: vec![Series::new(vec!["20210305", "20211024"]).into()],
-            func: CastFunction::create(DataType::Int32),
+            func: CastFunction::create("cast".to_string(), DataType::Int32),
             cast_type: DataType::Date32,
             expect: Series::new(vec![20210305i32, 20211024]),
```

Comment:
expect type is i32, but cast type is Date32

Reply:
Ok, The original code is  DateType::Int32..
I will change it into Date32.

Reply:
> expect type is i32, but cast type is Date32

DateType is 2020-03-05 not 20200305. 
The original code is Int32.. And it passed.